.github/workflows/publish-cli.yml

```diff
@@ -1,10 +1,6 @@
 name: Publish cli wallet
 
 on:
-  push:
-    branches:
-      - develop
-      - production
   workflow_dispatch:
```

Comment:
Can you di it one line for consistency pls `on: workflow_dispatch`

Reply:
That's not allowed `Nested mappings are not allowed in compact mappings`

Reply:
Nvm, forgot to remove `:`